pandas/tests/test_resample.py

```diff
@@ -2192,6 +2192,17 @@ def test_resample_datetime_values(self):
         res = df['timestamp'].resample('2D').first()
         tm.assert_series_equal(res, exp)
 
+    def test_apply_to_empty_series(self):
+        # GH 14313
+        series = self.create_series()[:0]
+
+        for freq in ['M', 'D', 'H']:
+            result = series.resample(freq).apply(lambda x: 1)
+            expected = series.resample(freq).apply(np.sum)
+
+            assert result.name == expected.name
```

Comment:
can you move this to ``Base`` instead to fully test Period/Timedelta Index (this will fail currently there as the code change is too specific)